airbyte-webapp/package.json

```diff
@@ -49,6 +50,7 @@
     "rehype-urls": "^1.1.1",
     "remark-frontmatter": "^4.0.1",
     "remark-gfm": "^3.0.0",
+    "rxjs": "^7.5.5",
```

Comment:
Very bold introduction of rxjs. Did you consider alternatives of implementing the similar behavior without observables?